File: 1.INICIANTE/EX_1012.c
#include<stdio.h>

double A=0, B=0, C=0, AREA=0;

main()
{
	scanf("%lf %lf %lf",&A, &B, &C);
	AREA = A*C/2;
	printf("TRIANGULO: %.3lf\n",AREA);	
	AREA = 3.14159*C*C;
	printf("CIRCULO: %.3lf\n",AREA);
	AREA = ((A+B)/2)*C;
	printf("TRAPEZIO: %.3lf\n",AREA);
	AREA = B*B;
	printf("QUADRADO: %.3lf\n",AREA);
	AREA = A*B;
	printf("RETANGULO: %.3lf\n",AREA);
	return 0;
	
}

File: 1.INICIANTE/EX_1012.py
a,b,c = input().split()
A = float(a)
B = float(b)
C = float(c)
tri = A*C/2
cir = 3.14159*C**2
trap = (A*C +B*C)/2
qua = B**2
ret = A*B
print(f'TRIANGULO: {tri:.3f}')
print(f'CIRCULO: {cir:.3f}')
print(f'TRAPEZIO: {trap:.3f}')
print(f'QUADRADO: {qua:.3f}')
print(f'RETANGULO: {ret:.3f}')